src/ontology/cob-edit.owl

```diff
@@ -56,12 +56,6 @@
     
 
 
-    <!-- http://purl.org/dc/elements/1.1/date -->
-
-    <owl:AnnotationProperty rdf:about="http://purl.org/dc/elements/1.1/date"/>
```

Comment:
Changed the dc elements namespace to dc terms throughout to satisfy a new, default QC check in ODK 1.5.